From 06e00ce656da2233a6bf36b50bdf079af5d2f383 Mon Sep 17 00:00:00 2001
From: Ilya Dryomov <ilya.dryomov@inktank.com>
Date: Fri, 5 Sep 2014 20:16:52 +0400
Subject: [PATCH 154/213] libceph: select CRYPTO_CBC in addition to CRYPTO_AES
References: fate#318918
Git-commit: dc220db03f15c9875aa09c36beba582f20c76be1
Patch-mainline: v3.18-rc1

We want "cbc(aes)" algorithm, so select CRYPTO_CBC too, not just
CRYPTO_AES.  Otherwise on !CRYPTO_CBC kernels we fail rbd map/mount
with

    libceph: error -2 building auth method x request

Signed-off-by: Ilya Dryomov <ilya.dryomov@inktank.com>
Acked-by: David Disseldorp <ddiss@suse.de>

---
 net/ceph/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/ceph/Kconfig b/net/ceph/Kconfig
index e50cc69..f8cceb9 100644
--- a/net/ceph/Kconfig
+++ b/net/ceph/Kconfig
@@ -3,6 +3,7 @@ config CEPH_LIB
 	depends on INET
 	select LIBCRC32C
 	select CRYPTO_AES
+	select CRYPTO_CBC
 	select CRYPTO
 	select KEYS
 	default n
-- 
2.1.4

